lib/services/geocoding.js

```diff
@@ -34,6 +34,8 @@ function roundTo(value, places) {
  * @param {string} options.types a comma seperated list of types that filter
  * results to match those specified. See https://www.mapbox.com/developers/api/geocoding/#filter-type
  * for available types.
+ * @param {number=5} options.limit is the maximum number of results to return, in range 1 to 10.
```

Comment:
Hmm, I copied the formatting of other params already there: `decimal degrees latitude, in range -90 to 90`, but agree it could be more clear. In another place I used `between 1 and 10 inclusive`.